sql/base/auth_database.sql

```diff
@@ -583,6 +583,7 @@ CREATE TABLE `realmlist` (
   `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `name` varchar(32) NOT NULL DEFAULT '',
   `address` varchar(255) NOT NULL DEFAULT '127.0.0.1',
+  `address6` varchar(255) NOT NULL DEFAULT '',
```

Comment:
The realm server returns both addresses to the client as separate lists, and the client makes the choice (there's a flag in the client under System/Network plus it probably does some local connectivity checks as well) - therefore the realmlist needs to know both.

It could I guess be done as a comma separated list but you have to then ensure there's only one of each and parse the strings rather than just chucking them at boost with the right flag set.